packages/core/src/react-integration/__tests__/useResource-endpoint.web.tsx

```diff
@@ -327,12 +343,20 @@ describe('useResource()', () => {
     const fetchKey = InvalidIfStaleArticleResource.detail().getFetchKey(
       payload,
     );
+    const entityMeta: any = {};
+    for (const k in entities) {
+      entityMeta[k] = {};
+      for (const pk in entities[k]) {
+        entityMeta[k][pk] = { date: 0, expiresAt: 0 };
```

Comment:
perhaps this repeated logic could be factored into a shared function?

Reply:
oh ya, i did that in another test file. i'm so liberal with copypasta in tests :P